packages/dbgpt-serve/src/dbgpt_serve/auth/models/models.py

```
@@ -0,0 +1,522 @@
+class AuditEventDao(BaseDao):
+    """Append-only persistence operations for audit events."""
+
+    _FILTER_FIELDS = {
+        "operator_user_id",
+        "target_account_set_id",
+        "target_type",
+        "target_id",
+        "action",
+        "result",
+        "request_id",
+    }
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)